web/views/style/create.tmpl

```diff
@@ -38,23 +38,22 @@
 <form class="{{ $method }}" method="post" action="/{{ $method }}" enctype="multipart/form-data">
 	<label for="name">Name</label>
 	<input
-		required pattern=".+"
+		required pattern=".{1,50}"
```

Comment:
For consistency should we move away from pattern to maxlength?

Reply:
This consistency will slightly decrease UX for noJS users and we will increase amount of JS we run. I prefer HTML-only solution when possible.

Reply:
Honestly, I think that would make sense since we're just checking the length of that input. NoJS users won't be losing out on a whole lot here.

Reply:
They're losing red border indication and that will require some of us to implement this.